bin/i18n/sync-in.rb

```diff
@@ -116,6 +116,13 @@ def get_i18n_strings(level)
     end
   end
 
+  if level.is_a? BubbleChoice
+    i18n_strings["sublevels"] = {}
+    level.sublevels.map do |sublevel|
+      i18n_strings["sublevels"][sublevel.name] = get_i18n_strings sublevel
```

Comment:
@daynew This will address your concern that translations should not depend on order of the sublevels. I've updated the PR description with a new output. 

Btw, `Level.name` is unique because of this validation: https://github.com/code-dot-org/code-dot-org/blob/772f5d54fa01beb06b29c93a4244609aaeacf52c/dashboard/app/models/levels/level.rb#L55